SU2_CFD/src/drivers/CDriver.cpp

```diff
@@ -2654,6 +2654,109 @@ void CDriver::Turbomachinery_Preprocessing(CConfig** config, CGeometry**** geome
 
 CDriver::~CDriver(void) {}
 
+void CDriver::Print_DirectResidual(RECORDING kind_recording) {
```

Comment:
I now made a general one for both: single and multizone. 
If this is not liked the commit can of course be reverted